.travis.yml

```diff
@@ -161,3 +164,6 @@ after_success:
         fi
       fi
   fi
+  if [[ $BUILD_DOCS == false && $TEST_ARGS != "--pep8-only" ]]; then
+     codecov
```

Comment:
Mostly just curious about this: How does this uploading work without credentials?  Is anyone able to just upload matplotlib coverage results?